server/ServerFSM.cpp

```diff
@@ -969,6 +986,8 @@ sc::result MPLobby::react(const LobbyUpdate& msg) {
     // if got important lobby changes so players shoul reset their ready status
     bool has_important_changes = false;
 
+    const GG::Clr CLR_NONE = GG::Clr(0, 0, 0, 0);
```

Comment:
I think there's already `GG::CLR_ZERO` defined for this...

Reply:
It requires to link to GG library.

Reply:
If you know how to initialize the constants in the header, sounds good. I just get errors that the expression must have a constant value. Not sure why my attempts don't.

Reply:
Pushed new commit with changes `GG::Clr` to have a constexpr constructor and used it in `ClrConstants.h`. If it will build without error I'll remove all other color constants from `ClrConstants.cpp`.

Reply:
I suppose `ClrConstants.cpp` should be removed from Xcode project as well.

Reply:
The manually maintained Xcode project has been removed quite a while ago, so no need fot that.